setup.py

```diff
@@ -229,15 +229,23 @@ def get_data_files():
 jupyter_client_req = 'jupyter_client>=5.3.1'
 
 extra_requirements = {
-    'test': ['pytest', 'pytest-cov', 'ipykernel', jupyter_client_req, 'ipywidgets>=7'],
+    'test': ['pytest',
+             'pytest-cov',
+             'pytest-dependency',
+             'ipykernel',
+             jupyter_client_req,
+             'ipywidgets>=7',
+             'pyppeteer==0.2.2',
```

Comment:
perhaps hoist this hard pin, like `jupyter_client`

Reply:
Sure, can do